src/site/stages/build/page-builder/schemas/page.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    title: {
+      $ref: 'GenericNestedString',
+    },
+    field_intro_text: {
+      $ref: 'GenericNestedString',
+    },
+    field_description: {
+      $ref: 'GenericNestedString',
+    },
+    field_featured_content: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_content_block: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_alert: {
+      type: 'array',
+    },
+    field_related_links: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_administration: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_page_last_built: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          value: { type: 'string', format: 'custom-date-time' },
```

Comment:
For situations like this would it be useful to have some helper that takes a `format` argument that it uses to modify a `genericNestedString` with?

Reply:
Yeah, we could do that if we run into this a lot. Alternatively, what we could do is create a new schema for the nested custom-date-time.